websites/W/Wikipedia/dist/metadata.json

```diff
@@ -14,15 +21,19 @@
     "zh_CN": "维基百科是一种多语言的在线百科全书，包含基于Wiki的内容的协作编辑系统。"
   },
   "url": "wikipedia.org",
-  "version": "2.6.4",
+  "version": "2.8.0",
   "logo": "https://i.imgur.com/zFdTVKT.png",
   "thumbnail": "https://i.imgur.com/vJiuzZy.png",
   "color": "#f9f9f9",
   "tags": [
     "encyclopedia",
+    "information",
     "popular",
-    "informations"
+    "wiki",
+    "wikimedia-foundation",
+    "wikimedia",
+    "wmf"
   ],
   "category": "other",
-  "regExp": "([a-z0-9-]+[.])*wikipedia[.]org[/]"
-}
\ No newline at end of file
+  "regExp": "[a-z0-9-]+[.]wikipedia[.]org[/]"
```

Comment:
regex didnt need changing

Reply:
Well, wikipedia.org is redirected to www.wikipedia.org, so I think it is okay to change it. LMK if I still need to change it.

Also, I will need to change the version from 2.8.0 to 2.7.0, because I skipped 2.7. I still need to figure it out if I need to bump to the minor one (2.7.0) or the major one (2.8.0).

Reply:
some of the pages have multiple first name thingys so like bla.bla.wikipedia.org

Reply:
Well, about Wikipedia (and the whole Wikimedia wikis), the subdomains are following the format [iso-639].wikipedia.org, so it's not going to have their own "sub-subdomains."

I could do it by using the + quantifier instead of *, LMK how you think about it.

Reply:
Never mind. I forgot there is the mobile version of the Wikipedia, like en.m.wikipedia.org. Whoops.

I changed it so it uses the + quantifier, along with additional support for the mobile site.

Reply:
+ = between 1 and unlimited times
* = between 0 and unlimited times

So if it doesnt have a subdomain the presence wont work, cause right now it must have 1.
not sure if wikipedia does **always** have atleast www. infront.

Reply:
> `+` = between 1 and unlimited times
> `*` = between 0 and unlimited times

dw I understand

> not sure if wikipedia does **always** have atleast www. infront.

Yes, it is, and I'm very certain. They can do it for something like [Wiksource](https://wikisource.org/wiki/Main_Page), but they didn't do it here, and it's out of the scope of this presence.